src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -278,4 +278,13 @@ public void setDatastorePath(File datastorePath) {
   public void setEnvironment(Map<String, String> environment) {
     this.environment = environment != null ? ImmutableMap.copyOf(environment) : null;
   }
+
+  @Override
+  public File getWorkingDirectory() {
+    return workingDirectory;
+  }
+
+  public void setWorkingDirectory(File workingDirectory) {
+    this.workingDirectory = workingDirectory;
```

Comment:
perhaps throw an exception if this isn't a directory?
should we use Path instead of File here? I don't know.